modules/tinymce/src/themes/silver/test/ts/headless/components/urlinput/UrlInputTest.ts

```diff
@@ -35,7 +35,7 @@ describe('headless.tinymce.themes.silver.components.urlinput.UrlInputTest', () =
           {
             type: 'header' as LinkTargetType,
             title: 'Header2',
-            url: '#h_2abefd32',
```

Comment:
This was quite weird and I think was probably a copy/paste accident at some point 🤷

Reply:
Yeah. Good change. I was keeping it because I thought maybe the intent was to clearly distinguish between the values we wanted and the ones we didn't. But copy/paste sounds just as likely.